docs/usage/configuration-options.md

```diff
@@ -69,7 +69,7 @@ With the above config:
 ## additionalBranchPrefix
 
 This value defaults to an empty string, and is typically not necessary.
-Some managers populate this field for historical reasons, for example we use `docker-` for Docker branches, so they may look like `renovate/docker-ubuntu-16.x`.
+Some managers previously populate this field, but they no longer do it by default.
```

Comment:
```suggestion
Some managers previously populated this field, but they no longer do so by default.
```

- The sentence is in the past tense, so `populate` should be `populated`.
- `do it` -> `do so` looks better to me.